stl/inc/queue

```diff
@@ -18,6 +18,7 @@
 
 #if _HAS_CXX23
 #include <__msvc_ranges_to.hpp>
+#include <format>

```

Comment:
Currently `<queue>` and `<stack>` are made to include `<format>`.
I'll make a follow-up PR to reduce inclusion from `<queue>`, `<stack>`, `<stacktrace>`, and `<thread>`.